### ISSUE TITLE
More detailed installation help?

### ISSUE THREAD

**Issue by jaas666** [sgupta999/GuptaSmartthingsRepository#1]
I'm new to all this. I got an EMU-2 hooked to a RPI 3 B+ and would love to get this going but don't know how.

Any chance we can get a detailed installation document?

Thanks in advance.

**Comment by jaas666** [comment]
Got it to work. I was missing the broker (mosquitto). Once installed it started workind… kind of.

I only have the current meter reading on my Device (in smartthings app). The graph doesn’t load (keeps spinning).

The info at the bottom for today, yesterday etc won’t load either.

Also it seems that consumption only updates when I change “pages” in smartthings… for example going to the list of devices and back to the meter device it’ll update. If I keep looking at it it won’t.

Hopefully this project isn’t abandoned.

**Comment by jaas666** [comment]
After a lot of time trying to understand how all of this worked I managed to find out why data wasn't populating.

My EMU2 device is new, therefore it still doesn't have "previous period data".
raven.js fires the mqtt payload after reading the previous period data from the EMU, since there's no data to deliver the mqtt msg is never sent.

What I did to solve the issue was to manually restart the period in the EMU device by going to the "total to date" page display on the meter and holding the UP button for 5 seconds. This will open a new period thus creating an previous period with whatever data you have till date.

After this the device was able to populate all the data without problem.
